### PR TITLE
Align issues with search bar and bottom row

### LINKED ISSUE #142: Alignment
image

Whole thing needs moving slightly to the right? @AlbertoElias

### PR DESCRIPTION
**Pull request by AlbertoElias**
Thanks to @onishiweb for finding the fix

Closes #142

### CODE REVIEW

File: src/scss/_nav.scss [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 	@include oGridRespondTo(M) {
 		position: relative;
+		left: 20px;
```
Comment (onishiweb) [inline src/scss/_nav.scss]: Would it be worth making this a variable? As it's the button size, which would mean it could be updated in multiple places.

**Comment by onishiweb** [comment]
Woop! Glad that worked. Just one comment/question for you and it's good to go.

File: src/scss/_nav.scss (hunk 1 above)
Comment (AlbertoElias) [inline src/scss/_nav.scss]: That value is just the default from o-ft-icons, so allowing to change that would mean support to change the icon itself, and I don't think that's something we should support. I believe the design is that it should always be that size

Comment (onishiweb) [inline src/scss/_nav.scss]: ok cool. I was probably looking at the compiled source when I saw the values being written. All good then

**Pull request merged**